mytk/labels.py

```diff
@@ -26,11 +27,14 @@ def set_label_wrap(self, event):
 
 class URLLabel(Label):
     def __init__(self, url=None, text=None):
+        # if text is None:
+        #     text = url
```

Comment:
commented out code

Reply:
maybe it shouldn't be commented out, because now if you don't give a text you won't have the url